packages/react/src/components/Switch/Switch.js

```diff
@@ -24,6 +25,16 @@ const Switch = props => {
     ...other
   } = props;
 
+  const tabRef = useRef();
+
+  useEffect(() => {
+    if (selected && tabRef.current) {
```

Comment:
```suggestion
  const tabRef = useRef(null);
```

Seems like the suggestion based on: https://reactjs.org/docs/hooks-reference.html#useref

Reply:
Whoops, wrong place for suggestion lol